data/migrations/V0044__auditsearch_view.sql

```diff
@@ -0,0 +1,96 @@
+SET search_path = auditsearch, pg_catalog;
+
+--
+-- 1)Name: cmte_audit_vw; Type: VIEW; Schema: auditsearch; Owner: fec
+--
+
+CREATE OR REPLACE VIEW cmte_audit_vw AS
+SELECT dc.cmte_id,
+    dc.cmte_nm,
+    dc.fec_election_yr,
+    dc.cmte_dsgn,
+    dc.cmte_tp,
+        CASE
+            WHEN dc.cmte_tp::text = 'H'::text THEN 'Congressional'::text
```

Comment:
I noticed we are naming committee types differently that the rest of the site I mad a small follow up task here https://github.com/18F/openFEC/issues/2827  There is so much good work in this PR I do not want to hold up the whole PR for a small thing.